test/math.jl

```diff
@@ -192,6 +192,25 @@ end
 @test exp2(Float16(2.)) ≈ exp2(2.)
 @test log(e) == 1
 
+@testset "check exp2(::Int) matches exp2(::Float)" begin
+    for ii in -2048:2048
+		expected = exp2(float(ii))
```

Comment:
search the git log for the last commit I made on master that removed a bunch of tabs, there's a grep command there that I use once in a while to find things that slip through. there are a few files that are either full of tabs that I haven't changed, or have a small number of them for testing purposes though. wouldn't be a bad idea to make check-whitespace check for more things though. better idea would be to replace it with a fix-whitespace script that we just run once a week or so.